index.js

```diff
@@ -243,6 +244,11 @@ exports.request = function (options, callback) {
         return finish();
     }
 
+    if(options.insecure && options.insecure === true) {
```

Comment:
Can you remove this if block? It shouldn't be necessary. Any options that aren't recognized are [passed to curl](https://github.com/node-js-libs/curlrequest/blob/30e2c373d047d2c9ca795408f76bd3c293793988/index.js#L223) as-is. If an option has `true` as a value, it's passed as a flag, e.g. `{insecure: true}` would be passed as `--insecure`.